### ISSUE TITLE
-q flag does not suppress update warnings

### ISSUE THREAD

**Issue by mattlqx** [ninxsoft/mist-cli#169]
### :beetle: Description

When using the quiet flag in conjunction with json output, the update banner still occurs if your version is out of date. This breaks json parsing. I would rather have the output be parsable and be out-of-date than it be broken immediately after an update is published.

### :clipboard: Steps to Reproduce

1. Run `mist list installer --include-betas -o json -q`
2. Observe...
```
┌──────────────────┐                                                                                                                                                                    
│ UPDATE AVAILABLE │                                                                                                                                                                    
└──────────────────┘                                                                                                                                                                    
  ├─ There is a mist update available (current version: 2.0, latest version: 2.1)                                                                                                       
  ├─ Visit https://github.com/ninxsoft/mist-cli to grab the latest release of mist                                                                                                      
[                                                                                                                                                                                       
  {                                                                                                                                                                                     
    "build" : "24A5279h",                                                                                                                                                               
    "compatible" : true,                                                                                                                                                                
    "date" : "2024-06-24",                                                                                                                                                              
    "identifier" : "062-22031",                                                                                                                                                         
    "name" : "macOS Sequoia Beta",                                                                                                                                                      
    "size" : 15007531436,                                                                                                                                                               
    "version" : "15.0"                                                                                                                                                                  
  },                         `
...
```

### :white_check_mark: Expected Behaviour

`-q` particularly in conjunction with a parsable format with `-o` should not produce output outside of that format.

### :computer: Environment

- mist version (`mist --version`): 2.0
- macOS Version (`sw_vers`): 23E224
- Hardware (`system_profiler SPHardwareDataType`):
  - Model Identifier: Macmini8,1
  - Chip : 6-Core Intel Core i7

**Comment by ninxsoft** [comment]
Nice catch! This is fixed in #170

FWIW, you can also export the list output using the `--export` flag:

```shell
  -e, --export <export>   Specify the path to export the list to one of the following formats:
                          * /path/to/export.csv (CSV file)
                          * /path/to/export.json (JSON file)
                          * /path/to/export.plist (Property List file)
                          * /path/to/export.yaml (YAML file)
                          Note: The file extension will determine the output file format.
```

**Examples:**

```shell
mist list firmware --export test.json
mist list installer --export test.yaml
```

**Comment by mattlqx** [comment]
Thanks Nindi!
